packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWForm.tsx

```diff
@@ -11,6 +11,8 @@ type FormProps = {
   className?: string;
   initialValues?: Object;
   validationSchema: z.Schema<any, any>;
+  watchlist?: string[];
```

Comment:
not sure, but do we need the `watchList`, its not used in the form

Reply:
thanks for catching this